lib/utils/constants.dart

```diff
@@ -1,4 +1,46 @@
+import '../flagship_version.dart';
+
 enum Mode {
   DECISION_API,
   BUCKETING,
 }
+
+/// START
+const String STARTED = "Flagship SDK (version: $FlagshipVersion) is started";
+const String INITIALIZATION_PARAM_ERROR = "Params 'envId' is not valide.";
+
+// ToDo: update this wording when the sdk manage other type
+
+// CONTEXT
+const String CONTEXT_UPDATE = "Update context with Map %s";
+const String CONTEXT_PARAM_ERROR =
+    "params 'key' must be a non null String, and 'value' must be one of the " +
+        "following types : String, Number, Boolean.";
+const String PARSING_ERROR = "an error occured whil parsing ";
+
+// MODIFICATION
+const String SYNCHRONIZE_MODIFICATIONS =
+    "Flagship SDK - Synchronize modifications.";
+const String SYNCHRONIZE_MODIFICATIONS_RESULTS =
+    "Flagship SDK - Synchronized modifications are %s";
+
+const String GET_MODIFICATION_ERROR =
+    "An error occured while retreiving modification for key '%s'. Default value is returned.";
+const String GET_MODIFICATION_INFO_ERROR = "No modification for key '%s'.";
+
+// HITS
+const String HIT_SUCCESS = "Flagship Hit sent with success";
+const String HIT_FAILED = "Flagship Failed to send hit";
```

Comment:
I feel the uppercase letters are a bit weird ? Why "Failed" and not "Failed". 
Would it be simpler if we put everything in lowercase (except maybe the name "Flagship")

Reply:
oaky